website/docs/r/packet_capture.html.markdown

```diff
@@ -0,0 +1,188 @@
+  resource_group_name  = "${azurerm_resource_group.test.name}"
+  target_resource_id   = "${azurerm_virtual_machine.test.id}"
+
+  storage_location {
+    storage_account_id = "${azurerm_storage_account.test.id}"
+  }
+
+  depends_on = ["azurerm_virtual_machine_extension.test"]
+}
+```
+
+~> **NOTE:** This Resource requires that [the Network Watcher Virtual Machine Extension](https://docs.microsoft.com/azure/network-watcher/network-watcher-packet-capture-manage-portal#before-you-begin) is installed on the Virtual Machine before capturing can be enabled which can be installed via [the `azurerm_virtual_machine_extension` resource](virtual_machine_extension.html).
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name to use for this Packet Capture. Changing this forces a new resource to be created.
+
+* `network_watcher_name` - (Required) The name of the Network Watcher. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which the Network Watcher exists. Changing this forces a new resource to be created.
+
+* `target_resource_id` - (Required) The ID of the Resource to capture packets from. Changing this forces a new resource to be created.
+
+~> **NOTE:** Currently only Virtual Machines ID's are supported.
+
+* `maximum_bytes_per_packet` - (Optional) The number of bytes captured per packet. The remaining bytes are truncated. Defaults to `0` (Entire Packet Captured). Changing this forces a new resource to be created.
+
+* `maximum_bytes_per_session` - (Optional) Maximum size of the capture in Bytes. Defaults to `10737441824` (1GB). Changing this forces a new resource to be created.
+
+* `maximum_capture_duration` - (Optional) The maximum duration of the capture session in seconds. Defaults to `18000` (5 hours). Changing this forces a new resource to be created.
+
+* `storage_location` - (Required) A `storage_location` block as defined below. Changing this forces a new resource to be created.
+
+* `filter` - (Optional) One or more `filter` blocks as defined below. Changing this forces a new resource to be created.
+
+---
+
+A `storage_location` block contains:
+
+* `file_path` - (Optional) A valid local path on the targeting VM. Must include the name of the capture file (*.cap). For linux virtual machine it must start with `/var/captures`.
+
+* `storage_account_id` - (Optional) The ID of the storage account to save the packet capture session
+
+~> **NOTE:** At least one of `file_path` or `storage_account_id` must be specified.
+
+A `filter` block contains:
+
+* `local_ip_address` - (Optional) The local IP Address to be filtered on. Notation: "127.0.0.1" for single address entry. "127.0.0.1-127.0.0.255" for range. "127.0.0.1;127.0.0.5"? for multiple entries. Multiple ranges not currently supported. Mixing ranges with multiple entries not currently supported. Changing this forces a new resource to be created.
```

Comment:
Extraneous `?` before for multiple entries

Reply:
that's actually a copy and paste from [the Microsoft Documentation](https://docs.microsoft.com/en-us/rest/api/network-watcher/packetcaptures/create)

<img width="807" alt="screen shot 2018-04-03 at 08 16 01" src="https://user-images.githubusercontent.com/666005/38232441-4e341ae4-3717-11e8-9f4f-1a4fd2c8a4af.png">

but yes, I'd agree let's remove it